test/metric_value_benchmark/emf_test.go

```diff
@@ -90,7 +100,52 @@ func (t *EMFTestRunner) validateEMFMetric(metricName string) status.TestResult {
 		return testResult
 	}
 
-	if !isAllValuesGreaterThanOrEqualToZero(metricName, values) {
+	if len(values) == 0 {
+		return testResult
+	}
+
+	for _, v := range values {
+		if v != 5 {
+			return testResult
+		}
+	}
+
+	testResult.Status = status.SUCCESSFUL
+	return testResult
+}
+
+func validateEMFLogs(group, stream string) status.TestResult {
+	testResult := status.TestResult{
+		Name:   "emf-logs",
+		Status: status.FAILED,
+	}
+
+	rs := jsonschema.Must(schema)
+
+	now := time.Now()
+	ok, err := awsservice.ValidateLogs(group, stream, nil, &now, func(logs []string) bool {
+		if len(logs) < 1 {
+			return false
+		}
+
+		for _, l := range logs {
+			keyErrors, e := rs.ValidateBytes(context.Background(), []byte(l))
+			if e != nil {
+				log.Println("failed to execute schema validator:", e)
+				return false
+			} else if len(keyErrors) > 0 {
+				log.Printf("failed schema validation: %v\n", keyErrors)
+				return false
+			}
+
+			if !strings.Contains(l, "\"EMFCounter\": 5") {
```

Comment:
```suggestion
			if !strings.Contains(l, "\"CloudWatchMetrics\"") {
```
in case of changing EMF format in the future IMO.

Reply:
No, the point is validating the metric value that is in the log.